io.itch.domestique-baston.yml

```diff
@@ -0,0 +1,19 @@
+app-id: io.itch.domestique-baston
```

Comment:
this app-id is not acceptable as you don't own `io.itch`. Like for github, you need the user/org. There are other `io.itch` based packages that do that.

Reply:
So the app-id should be `io.itch.domestique-baston.domestique-baston`?